app/src/main/java/org/kiwix/kiwixmobile/wifi_hotspot/WifiHotspotManager.java

```diff
@@ -0,0 +1,90 @@
+package org.kiwix.kiwixmobile.wifi_hotspot;
+
+import android.net.wifi.WifiConfiguration;
+import android.net.wifi.WifiManager;
+import android.os.Build;
+import android.os.Handler;
+import android.util.Log;
+import androidx.annotation.NonNull;
+import androidx.annotation.RequiresApi;
+import javax.inject.Inject;
+
+/**
+ * WifiHotstopManager class makes use of the Android's WifiManager and WifiConfiguration class
+ * to implement the wifi hotspot feature.
+ * Created by Adeel Zafar on 28/5/2019.
+ */
+
+public class WifiHotspotManager {
+  private static final String TAG = "WifiHotspotManager";
+  private WifiManager.LocalOnlyHotspotReservation hotspotReservation;
+  private HotspotStateListener hotspotStateListener;
```

Comment:
Can we inject this as well?

Reply:
Fixed d9b191bc784b47183ada993d9a54afcb56bf3624